spec/databases/sqlserver/schema/schema.sql

```diff
@@ -0,0 +1,15 @@
+IF NOT EXISTS (SELECT * FROM sys.tables WHERE name = N'users' AND type = 'U')
```

Comment:
Since you have moved the truncate tasks into the API. Do you still need this file?